internal/server/worker.go

```diff
@@ -190,19 +189,19 @@ func (w *Worker) clone() *Worker {
 
 // ActiveConnectionCount is the current number of sessions this worker is handling
 // according to the controllers.
-func (w *Worker) ActiveConnectionCount() uint32 {
-	return w.activeConnectionCount
+func (w *Worker) GetActiveConnectionCount() uint32 {
+	return w.ActiveConnectionCount
```

Comment:
Do we still need this accessor method? Instead can the worker service just access ActiveConnectionCount directly?

Reply:
Good call, thanks!